in_toto/model_test.go

```diff
@@ -1706,45 +1719,6 @@ func TestMetadataNoTime(t *testing.T) {
 	})
 }
 
-// Verify that the behaviour of definedInMaterial can be controlled,
-// as there is a semantic difference in value present or 0.
-func TestRecipe(t *testing.T) {
```

Comment:
Ooh, this was missed with the v0.1 change. Thanks!

Reply:
I actually removed this test because it was testing `definedInMaterial` in `recipe`, which was removed in `v0.2`

Reply:
Yep! But with the move to the v0.1 package, this test was duplicated:

https://github.com/in-toto/in-toto-golang/blob/92ba2b6505ff47deeae191aaf38b8b31a7f0725a/in_toto/model_test.go#L1711-L1746

and

https://github.com/in-toto/in-toto-golang/blob/92ba2b6505ff47deeae191aaf38b8b31a7f0725a/in_toto/slsa_provenance/v0.1/provenance_test.go#L154-L189

No harm done though :)